index.html

```diff
@@ -918,15 +918,21 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the payment process itself, but recording the fact of payment with a web
```

Comment:
How about:

NFC can have multiple records that are read when touched, and Web NFC actions can be taken at the same time with the payment that may be implemented using native NFC technology.